packages/blocks/src/database-block/table/clipboard.ts

```diff
@@ -49,7 +47,12 @@ export class TableViewClipboard implements BaseViewClipboard {
       uiEventDispatcher.add(
         'copy',
         ctx => {
+          const selection = getDatabaseSelection(this._root);
+          const tableSelection = selection?.getSelection('table');
+          if (!tableSelection) return false;
+
           this._onCopy(ctx);
```

Comment:
@Saul-Mirone 

This event handler contains an asynchronous operation.
In the current scenario, if it is a copy inside the database, `return true` can be used to prevent bubbling.